project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml"
 addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"     % "3.9.0")
```

Comment:
sbt-auto-build -> 3.15.0
play -> 2.8.21
swap sbt-scalastyle for "org.scalastyle"    % "scalastyle-sbt-plugin" % "1.0.0" as in PR bot comments
sbt-scalafmt -> 2.5.2